src/entitysdk/models/contribution.py

```diff
@@ -42,6 +41,12 @@ class Contribution(Identifiable):
         ),
     ]
     entity: Annotated[
-        Entity | None,
+        "Entity | None",
```

Comment:
Could you explain why you turned `Entity | None` into a string?

Reply:
It's because Entity and Contribution have references on each other. Using postponed evaluation of the types you can declare a forward ref here which is evaluated at the end using the model_rebuild without encountering import cycles.

Reply:
ok I see, thanks for the explanations